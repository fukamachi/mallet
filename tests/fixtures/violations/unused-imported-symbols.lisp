;; Test file for unused-imported-symbols rule

(defpackage #:test-unused-imports
  (:use #:cl)
  (:import-from #:alexandria
                #:hash-table-keys
                #:hash-table-values  ; Unused
                #:hash-table-count)  ; Unused
  (:export #:load-config))
(in-package #:test-unused-imports)

(defun load-config ()
  (hash-table-keys (make-hash-table)))
