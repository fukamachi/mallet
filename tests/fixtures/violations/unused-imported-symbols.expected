# Expected violations for unused-imported-symbols.lisp
# Format: line:column rule-name severity

7:16 unused-imported-symbols info
8:16 unused-imported-symbols info
